docs/guides/apps_marketplace.md

```diff
@@ -101,25 +101,29 @@ This resource has the following arguments definition:
 * `disable_hooks` - (Optional) Disable app v2 chart hooks. Default: `false` (bool)
 * `disable_open_api_validation` - (Optional) Disable app V2 Open API Validation. Default: `false` (bool)
 * `force_upgrade` - (Optional) Force app V2 chart upgrade. Default: `false` (bool)
-* `wait` - (Optional) Wait until app is deployed. Default: `false` (bool)
+* `wait` - (Optional) Wait until app is deployed. Default: `true` (bool)
 * `annotations` - (Optional/Computed) Annotations for the app v2 (map)
 * `labels` - (Optional/Computed) Labels for the app v2 (map)
 
 ### Examples
 
-These are some examples how to deploy some Rancher cerfified apps v2:
+These are some examples how to deploy some Rancher cerfified Apps v2:
 
-* Deploy Rancher monitoring
+**Note** Apps v2 certified by Rancher should be installed within fixed app name and namespace defined by chart annotations, `catalog.cattle.io/release-name` and `catalog.cattle.io/namespace`. The provider will be forcing them on Rancher cerfified Apps v2 installation, but generating a diff on next `terraform apply`, if app v2 name and/or namespace is customized by user. Adjusting the values at tf file should supress the diff
+
+**Note** Latest app v2 version will be deployed, if `chart_version` is not provided
+
+* `rancher-monitoring` - Deploy Rancher monitoring
 
 ```hcl
 resource "rancher2_app_v2" "rancher-monitoring" {
-  cluster_id = "&lt;CLUSTER_ID&gt;"
+  cluster_id = "<CLUSTER_ID>"
   name = "rancher-monitoring"
   namespace = "cattle-monitoring-system"
   repo_name = "rancher-charts"
   chart_name = "rancher-monitoring"
   chart_version = "9.4.200"
-  values = &lt;&lt;EOF
+  values = <<EOF
```

Comment:
This is not valid example for a terraform here doc. Might be better to provide a working example such as:
```
values = yamlencode(
        {
          "map " = {
            "key1"      = "value"
            "key2" = var.key2.vaue
          }
        }
)
```